### PR TITLE
feat(financial aid): Adding Web Veita to tsconfig and Identicon package

### PR DESCRIPTION
**Pull request by MargretFinnboga**
## What

Needed to add Web Veita to tsconfig for imports
Added Identicon package for Web Veita to create unique profile images for applicants 

## Why
tsconfig: clarity in code
package: for better visualization experience for staff working with the applications  

## Screenshots / Gifs

Screenshot 2021-06-23 at 15 00 56:

## Checklis

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

**Review by addi — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -327,6 +328,7 @@
     "@types/express": "4.17.0",
     "@types/folder-hash": "3.3.0",
     "@types/http-proxy-middleware": "0.19.3",
+    "@types/identicon.js": "^2.3.0",
```
Comment (eirikurn) [inline package.json:331]: Please remove the ^

File: package.json [hunk 2]
```diff
@@ -158,6 +158,7 @@
     "graphql-type-json": "0.3.2",
     "http-proxy-middleware": "0.19.1",
     "hypher": "0.2.5",
+    "identicon.js": "^2.3.3",
```
Comment (eirikurn) [inline package.json:161]: Please remove the ^

**Review by eirikurn — APPROVED** [review]
LGTM. It might be good to open a follow up discussion in the design discipline, if we want to use identicon in our design system for this use case (or if there are specific settings or alternatives we want to use).

**Review by eirikurn — CHANGES_REQUESTED** [review]
Sorry. Forgot the dep versions. Please fix those before merging :)

**Review by eirikurn — APPROVED** [review]

**Pull request merged**